### PR TITLE
Support Google Colaboratory

### LINKED ISSUE #99: Incompatible with Google Colaboratory
### Bug Description
As stated [here](https://stackoverflow.com/questions/51835926/google-colab-runtime-dies-when-using-autokeras), Autokeras kills the Google Colaboratory kernel, in CPU or GPU mode.

### Reproducing Steps
Run the following [notebook](https://colab.research.google.com/drive/1REPj8pVSpSUvvufHBY693RmkXjnBSKcg), which is a simple copy-paste of the AutoKeras tutorial, with a slight modification: in the AutoKeras API, the package autokeras.image_classifier seems to have been renamed to autokeras.image_supervised, and the [documentation](https://autokeras.com/start/#example) is not yet up to date.

### Expected Behavior
It should run, since Colaboratory runs Python 3.6

### Additional context
Colaboratory seems incompatible with the Python multiprocessing API. 

I modified the code of search.py/Searcher in order to remove it, and it worked.
More clearly, I replaced this:
` multiprocessing.set_start_method('spawn', force=True)`
`pool = multiprocessing.Pool(1)`
`train_results = pool.map_async(train, [(graph, train_data, test_data, self.trainer_args,
                                                os.path.join(self.path, str(model_id) + '.png'),
                                                self.metric, self.loss, self.verbose)])`        
By this:
`
        train_results = train((graph, train_data, test_data, self.trainer_args,
                                                os.path.join(self.path, str(model_id) + '.png'),
                                                self.metric, self.loss, self.verbose))
`

Maybe AutoKeras could have an option in order to desactivate the use of that API ?

### PR DESCRIPTION
**Pull request by GitHub30**
This pull request resolves #99 .

### CODE REVIEW

**Comment by GitHub30** [comment]
Sorry
